core/src/test/scala/integration/kafka/api/SaslSslAdminClientIntegrationTest.scala

```diff
@@ -18,6 +18,8 @@ import java.util
 import kafka.security.auth.{All, Allow, Alter, AlterConfigs, Authorizer, ClusterAction, Create, Delete, Deny, Describe, Group, Operation, PermissionType, SimpleAclAuthorizer, Topic, Acl => AuthAcl, Resource => AuthResource}
 import kafka.server.KafkaConfig
 import kafka.utils.{CoreUtils, JaasTestUtils, TestUtils}
+import kafka.utils.TestUtils._
```

Comment:
nit: seems we didn't need this import?

Reply:
above import is for using TestUtils.assertFutureExceptionTypeEquals method